lib/ActivityStream.jsm

```diff
@@ -236,6 +236,7 @@ const PREFS_CONFIG = new Map([
       const isEnabled = IS_NIGHTLY_OR_UNBRANDED_BUILD && locales && locales.includes(locale);
       return JSON.stringify({
         api_key_pref: "extensions.pocket.oAuthConsumerKey",
+        collapsible: true,
```

Comment:
Making collapsible false will still keep TopSites collapsible and will only remove section menu options and collapsible arrow from discovery stream top stories

Reply:
Yup. And probably same thing for highlights when that gets restored for parity.